src/CoreShop2VueStorefrontBundle/Bridge/Cart/CartContext.php

```diff
@@ -78,13 +79,21 @@ public function getCart()
         return $cart;
     }
 
+    /**
+     * @param StoreInterface    $store
+     * @param CustomerInterface $customer
+     *
+     * @return null|CartInterface
+     */
     public function findLatestByStoreAndCustomer(StoreInterface $store, CustomerInterface $customer)
     {
         $list = $this->cartRepository->getList();
-        $list->setCondition('customer__id = ? AND store = ? AND order__id is null ', [$customer->getId(), $store->getId()]);
-        $list->setOrderKey('o_id');
-        $list->setOrder('DESC');
-        $list->load();
+        $list
```

Comment:
isn't there a repository function for just exactly that?

Reply:
You're right, however I need those carts sorted by `o_id`, because `o_creationDate` can be tricky if you have two carts created with same timestamp.

Reply:
👍